regression/goto-instrument/typedef1/test.desc

```diff
@@ -0,0 +1,7 @@
+CORE
+main.c
+--dump-c
+^EXIT=0$
+^SIGNAL=0$
```

Comment:
It isn't clear to me what this test actually checks - could you perhaps add some matching lines